extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/AuthenticationProviderService.java

```diff
@@ -306,17 +311,43 @@ public LDAPAuthenticatedUser authenticateUser(Credentials credentials)
         }
 
     }
+    /**
+     * Returns parameter current ldap domain token generated from user credentials
+     * If no multiple LDAP are configured on GUACAMOLE_HOME such ldap-servers.yaml,
+     * a null is returned.
```

Comment:
Is this actually the way it works - if multiple LDAP servers/domains are not enabled, will it _always_ return `null`? Or are there situations where a single LDAP configuration could still result in a value being returned by this function - such as if the `userPrincipalName` attribute is used when talking to Active Directory?

Reply:
Yep, I have updated code comments in code to be more clear as it works when there are situations where single LDAP configuration could return domain name if exists otherwise null.